### ISSUE TITLE
Approach for Many-to-Many association with extra column

### ISSUE THREAD

**Issue by joshbedo** [balderdashy/sails-postgresql#206]
I've read through quite a few stack overflow posts and github issues but i can't seem to figure out how to do join tables using `through` on your model. I was wondering if that functionality exists in Sails 0.11.0 and also if there is any documentation. 

Thanks!

**VenueGroupVenue Model**
```
module.exports = {
  attributes: {
    venueGroup_id: {
      model: 'venueGroup',
      columnName: 'venueGroup_id',
      foreignKey: true,
      on: 'id'
    },

    venue_id: {
      model: 'venue',
      columnName: 'venue_id',
      foreignKey: true,
      on: 'id'
    },
....
```

**Venue Model**
```
        venueGroups: {
            collection: 'venueGroup',
            dominant: true,
            through: 'venuegroupvenue'
        },
```

**VenueGroup Model**
```
        venues: {
            collection: 'venue',
            through: 'venuegroupvenue'
        },
```

**Comment by joshbedo** [comment]
*bump*

**Comment by GeorgeSapkin** [comment]
AFAIK, `through` is not fully working ATM so you're better off using One-to-Many Many-to-One with `VenueGroupVenue` as the DIY junction table.

**Comment by devinivy** [comment]
@particlebanana and @atiertant have made good progress on making through-tables production ready and fully-featured.  They're getting there.  For now, you can use the technique described here: https://github.com/balderdashy/waterline/issues/705#issuecomment-60945411

**Comment by sailsbot** [comment]
Thanks for posting, @joshbedo.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
